fast64_internal/sm64/sm64_geolayout_writer.py

```diff
@@ -433,22 +428,49 @@ def convertArmatureToGeolayout(armatureObj, obj, convertTransformMatrix, camera,
             None,
             None,
             None,
-            meshGeolayout.nodes[i],
+            i,
             [],
             name,
             meshGeolayout,
             geolayoutGraph,
             infoDict,
             convertTextureData,
         )
+
+    def walk(node, fMeshes):
+        base_node = node.node
+        if type(base_node) == JumpNode:
```

Comment:
I was wondering, if this better than `isinstance()`? looks like you're using it later, I think one of both should be changed for consistency